egs2/lrs2/lipreading1/db.sh

```diff
@@ -0,0 +1 @@
+../../TEMPLATE/asr1/db.sh
```

Comment:
Seems that you forgot to update db.sh for this corpus?

Reply:
Thanks for you reminding! It's Done, as well as the egs2/readme.